you_got_listed.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version     = '>= 2.0.0'
 
-  s.add_dependency "activesupport", "~> 3"
+  s.add_dependency "activesupport", "~> 4"
```

Comment:
@pdornfel  You should use the `">= 3"` so that it works with our current rails version